src/java.base/share/classes/sun/nio/ch/IOUtil.java

```diff
@@ -47,15 +50,30 @@ static int write(FileDescriptor fd, ByteBuffer src, long position,
                      NativeDispatcher nd)
         throws IOException
     {
-        return write(fd, src, position, false, -1, nd);
+        return write(fd, src, position, false, false, -1, nd);
+    }
+
+    static int write(FileDescriptor fd, ByteBuffer src, long position,
+                     boolean async, NativeDispatcher nd)
```

Comment:
UnixAsynchronousSocketChannel is an outlier user of IOUtil and I think we need to see if we can avoid extending this class to async if possible.